src/Nethermind/Nethermind.Core/BlockHeaderProof.cs

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
```

Comment:
```suggestion
// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
```
use up to date year for new files